arangodb-net-standard.Test/AnalyzerApi/AnalyzerApiClientTest.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using ArangoDBNetStandard;
+using ArangoDBNetStandard.AnalyzerApi;
+using ArangoDBNetStandard.AnalyzerApi.Models;
+using ArangoDBNetStandard.Transport;
+using Moq;
+using Xunit;
+
+namespace ArangoDBNetStandardTest.IndexApi
+{
+    public class AnalyzerApiClientTest : IClassFixture<AnalyzerApiClientTestFixture>, IAsyncLifetime
+    {
+        private AnalyzerApiClient _analyzerApi;
+        private ArangoDBClient _adb;
+
+        public AnalyzerApiClientTest(AnalyzerApiClientTestFixture fixture)
+        {
+            _adb = fixture.ArangoDBClient;
+            _analyzerApi = _adb.Analyzer;
+        }
+
+        public Task InitializeAsync()
+        {
+            return Task.CompletedTask;
+        }
+
+        public Task DisposeAsync()
+        {
+            return Task.CompletedTask;
+        }
+
+        [Fact]
+        [Trait("Feature", "Analyzer")]
+        public async Task GetAllAnalyzersAsync_ShouldSucceed()
+        {
+            var res = await _analyzerApi.GetAllAnalyzersAsync();
+            Assert.Equal(HttpStatusCode.OK, res.Code);
+            Assert.False(res.Error);
+        }
+
+        [Fact]
+        [Trait("Feature", "Analyzer")]
+        public async Task PostAnalyzerAsync_ShouldSucceed()
```

Comment:
This test covers the successful path of the endpoint method which is great. It would be nice to also have one test to cover the failure path for each endpoint. Not a blocker though.

Example in this case, `PostAnalyzerAsync_ShouldThrow_WhenErrorIsReturned` or `PostAnalyzerAsync_ShouldThrow_WhenAnalyzerIsInvalid`.

Reply:
This is a good idea. Thanks. I have added the test.